one-ui/src/main/java/com/marklogic/hub/oneui/controllers/SavedQueriesController.java

```diff
@@ -0,0 +1,47 @@
+package com.marklogic.hub.oneui.controllers;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.marklogic.hub.oneui.managers.SavedQueriesManager;
+import com.marklogic.hub.oneui.models.HubConfigSession;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Scope;
+import org.springframework.context.annotation.ScopedProxyMode;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@Controller
+@RequestMapping(value = "/api/savedQueries")
+public class SavedQueriesController {
+
+    @Autowired
+    private SavedQueriesManager savedQueriesManager;
+
+    @Autowired
+    private HubConfigSession hubConfig;
+
+    @Bean
```

Comment:
I don't think Bean is buying you anything here, right? I think the method works the same without these two annotations on it. I recommend removing them to avoid any confusion.